subsys/regulator/Kconfig

```diff
@@ -0,0 +1,33 @@
+menuconfig REGULATOR
+	bool "Regulator"
+	help
```

Comment:
Should this have a default value. Same for the PI_REG. Is there any point of setting only the REGULATOR to true. The CMakeLists.txt will then be included but none of the files in it will be added.